packages/dante3/src/plugins/tipTapPlaceholder.ts

```diff
@@ -5,12 +5,14 @@ import { Plugin } from "prosemirror-state";
 export const Placeholder = Extension.create({
   name: "placeholder",
 
-  addOptions: {
-    emptyEditorClass: "is-editor-empty",
-    emptyNodeClass: "is-empty",
-    placeholder: "Write something …",
-    showOnlyWhenEditable: true,
-    showOnlyCurrent: true,
+  addOptions() {
```

Comment:
Why was this converted into a function?

Reply:
It has something to do with the internals of tiptap's type inference. It's how they recommend doing it in the typescript compatibility section of their docs and it was causing an error to leave it as-is

Reply:
ok, 👌